### PR TITLE
fix: faq not working

### LINKED ISSUE #5140: FAQ not working
### What happened?

After clicking on [FAQ](https://kodadot.xyz/rmrk/faq), it loaded an empty page. At this point, I'm not sure if this is a bug or if this was empty the whole time. 

### Please reproduce in steps

Scroll down, click on FAQ in the footer.

### Expected Behavior

open FAQ

### Screenshots

Screenshot 2023-03-01 at 12 46 19

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Gavin-Gong**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [X] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [X] Closes #5140
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality
- [X] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [X] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer?target=GqheyaQ3c8SFBJYorTXyLW6jvLXKNAXzbad77NXoLYJBXXs)

#### Community participation

- [X] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
or instead add
```js
      {
        path: '~/components/common',
        extensions: ['vue'],
      },
```
in `nuxt.config.js`

**Review by Jarsen136 — CHANGES_REQUESTED** [review]
Let's change the faq page path from `/rmrk/faq` to `/faq`

**Comment by roiLeo** [comment]
> Let's change the faq page path from `/rmrk/faq` to `/faq`

not yet this page is only related with rmrk chain for now later #4553

File: components/TheFooter.vue [hunk 1]
```diff
@@ -97,7 +97,7 @@ const menu: Menu[] = [
   },
   {
     name: $i18n.t('faq.page'),
-    url: '/rmrk/faq',
+    url: '/faq',
```
Comment (Jarsen136) [inline components/TheFooter.vue:100]: Sorry, as the review comment mentioned, we do not need to change the path for now.

Comment (Gavin-Gong) [reply components/TheFooter.vue]: > Sorry, as the review comment mentioned, we do not need to change the path for now.

😆 Done

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 37.7 USD/KSM ~ 0.531 $KSM
🧗 [GqheyaQ3c8SFBJYorTXyLW6jvLXKNAXzbad77NXoLYJBXXs](https://kusama.subscan.io/account/GqheyaQ3c8SFBJYorTXyLW6jvLXKNAXzbad77NXoLYJBXXs)
🔗 [0xece9e903f0b2250f395a71d42dc53665cbb531d6f107fc7b700805e5ee1aea8e](https://kusama.subscan.io/extrinsic/0xece9e903f0b2250f395a71d42dc53665cbb531d6f107fc7b700805e5ee1aea8e)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**